AIDriver.lua

```diff
@@ -2268,6 +2270,17 @@ function AIDriver:closeCovers(object)
 	end
 end
 
+---  This function fixes cover handling in MP.
+local function coverFixMP(self,superFunc,state,noEventSend)
```

Comment:
nitpicking, but the good practice is to start function names with a verb, so fixCoverMP() would be better (especially that cover is also a verb :)